src/Sarif/Autogenerated/Result.cs

```diff
@@ -51,7 +51,7 @@ public SarifNodeKind SarifNodeKind
         /// <summary>
         /// The index within the run.tool.extensions array of the tool component object which describes the plug-in or tool extension that produced the result.
         /// </summary>
-        [DataMember(Name = "extensionIndex", IsRequired = false, EmitDefaultValue = false)]
+        [DataMember(Name = "ruleExtensionIndex", IsRequired = false, EmitDefaultValue = false)]
```

Comment:
>ruleExtensionIndex [](start = 28, length = 18)

could not see a reason why the dataMember name should be different so i changed it to match the auto-gen value. if this was done for a specific reason, let us document with a comment.